pkgs/by-name/fy/fyne/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+
+  dbus,
+  libGL,
+  libX11,
+  libXcursor,
+  libXinerama,
+  libXi,
+  libXrandr,
+  libXxf86vm,
+  pkg-config,
+}:
+
+let
+  version = "2.4.4";
```

Comment:
I'd recommend use `buildGoModule (finalAttrs: {})` instead, or at least use `rec`. This may not work with r-ryantm.

Reply:
Instead of using the `let version ...` here? Sure, I can do that.